port/action/schema.go

```diff
@@ -461,7 +460,7 @@ func StringPropertySchema() schema.Attribute {
 			Optional:            true,
 		},
 		"format": schema.StringAttribute{
-			MarkdownDescription: "The format of the string property",
+			MarkdownDescription: "The format of the string property. Common accepted values include `email`,`hostname`,`ipv4`,`ipv6`,`uuid`,`uri`,`date-time`",
```

Comment:
i am not familiar with those kind of formats, can you refer me to the related docs?

Reply:
I used the JSON Schema Documentation, You can find more information on these formats [here](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1).

Reply:
The format property in port doesn't actually comply with the official json-schema format; we have our own formats which we've added (such as 'entity')

Please refer to [port's docs](https://docs.getport.io/actions-and-automations/create-self-service-experiences/setup-ui-for-action/user-inputs/#special-string-formats) for the available formats

Reply:
okay got it.. thanks